templates/repo/issue/view_content/reactions.tmpl

```diff
@@ -0,0 +1,32 @@
+{{if .Reactions}}
+	{{range $key, $value := .Reactions}}
+		<a class="ui label basic{{if $value.HasUser $.ctx.SignedUserID}} blue{{end}} has-emoji">
+			{{if eq $key "hooray"}}
+				:tada:
```

Comment:
There are no js handlers for clicking on these

Reply:
Yes, unreacting is not yet implemented :)